cxplat/cxplat_test/cxplat_test.vcxproj

```diff
@@ -131,7 +131,9 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="cxplat_memory_test.cpp" />
+    <ClCompile Include="cxplat_rundown_test.cpp" />
```

Comment:
As more and more stuff is added, it's going to make it an even bigger job to bring Posix platform support, which is a requirement for MsQuic, into the picture here. Perhaps it's time to do this so that future PRs can then add the piecemeal support for the platform.

Reply:
feel free :)
Should this move to the cxplat repo before then or after then?

Reply:
Let's move it to cxplat first.